src/Test.elm

```diff
@@ -0,0 +1,303 @@
+module Test exposing (Test, FuzzOptions, describe, test, filter, concat, fuzz, fuzz2, fuzz3, fuzz4, fuzz5, fuzzWith)
+
+{-| Writing tests.
+
+@docs Test, test
+
+## Organizing Tests
+
+@docs describe, concat, filter
+
+## Fuzz Testing
+
+@docs fuzz, fuzz2, fuzz3, fuzz4, fuzz5, fuzzWith, FuzzOptions
+-}
+
+import Random
+import Test.Internal as Internal
+import Expect exposing (Expectation)
+import Random exposing (Generator)
+import Fuzz exposing (Fuzzer)
+
+
+{-| A test which has yet to be evaluated. When evaluated, it produces one
+or more [`Expectation`](../Expect#Expectation)s.
+
+See [`test`](#test) and [`fuzz`](#fuzz) for some ways to create a `Test`.
+-}
+type alias Test =
+    Internal.Test
+
+
+{-| Run the given tests in order.
+
+    concat [ testDecoder, testSorting ]
+-}
```

Comment:
I agree with all that. Here is my understanding: the results would be displayed in-order, even though the evaluation may be interleaved. If true, that seems ideal / I feel like pointing out that they run in-order draws attention to something everyone would *expect* is the default anyway, making it feel weird to mention.